documentation/library_guide/parallel_api/iterators.rst

```diff
@@ -69,10 +69,18 @@ header.  All iterators are implemented in the ``oneapi::dpl`` namespace.
 * ``transform_iterator``: an iterator defined over another iterator whose dereferenced value is the result
   of a function applied to the corresponding element of the original iterator. Both the type of the original
   iterator and the unary function applied during dereference operations are required template parameters of
-  the ``transform_iterator`` class. The ``transform_iterator`` class provides three constructors:
+  the ``transform_iterator`` class. 
+
+  The unary functor provided to a transform iterator should have a const call operator that takes a single argument
+  whose parameter type can be converted to from the reference type of the original iterator. The functor's call operator
```

Comment:
Should the `can be converted to from the reference type` here be just `to`, `from`, or `to and from`?

Reply:
reworded for clarity to :
"The unary functor provided to a transform iterator should have a const call operator which is callable with an
  argument of the reference type of the original iterator."